packages/core-modules/contracts/storage/ElectionStorage.sol

```diff
@@ -68,7 +68,11 @@ contract ElectionStorage {
         /**
          * @dev Voting data for the current election
```

Comment:
This documentation may need a bit of tuning:
"Historical voting data" or something more semantically aligned.

Reply:
Fixed